packages/fxa-react/components/Portal/index.test.tsx

```diff
@@ -44,3 +48,35 @@ it('renders multiple instances with the same ID to the same DOM parent', () => {
   expect(fooPortal).toContainElement(getByTestId('p2'));
   expect(fooPortal).not.toContainElement(getByTestId('p3'));
 });
+
+it('applies a11y improvements when id is set to "modal"', () => {
+  document.body.innerHTML = `
+    <div data-testid="root"></div>
+    <div data-testid="adjacent-to-root"></div>
+  `;
+
+  const headerId = 'some-header-id';
+  const descId = 'some-desc-id';
+
+  const { getByTestId } = render(
+    <Portal id="modal" {...{ headerId }} {...{ descId }}>
+      <div>Hi mom</div>
+    </Portal>
+  );
+
+  const body = document.body;
+  const root = getByTestId('root');
+  const adjacentToRoot = getByTestId('adjacent-to-root');
+  const modal = body.querySelector('#modal');
+
+  expect(body.classList.contains('overflow-hidden')).toBe(true);
```

Comment:
This might be simplified with `expect(body.classList).toContain('overflow-hidden')`